converter/static/js/index.js

```diff
@@ -112,7 +115,7 @@ Array.prototype.forEach.call( forms, function( form )
                     var result_text = result_data.result;
                     document.title = "Result Page";
                     document.getElementById('the_title').innerHTML = "Result";
-                    document.getElementById('result_content').innerHTML = result_text;
+                    //document.getElementById('result_content').innerHTML = result_text;
```

Comment:
Is this meant to be deleted? or commented for debugging and forgetting to revert?

Reply:
deleted